guides/common/modules/proc_rex-pull-based-performance-impact-considerations.adoc

```diff
@@ -0,0 +1,24 @@
+[id="REX_pull_based_performance_impact_considerations_{context}"]
+= REX pull-based transport performance impact considerations
+
+When {ProjectServer} is configured to use REX pull-based transport for remote execution jobs using "Script" provider, {SmartProxyServer} sends a notification about new job to the client machines via MQTT. This notification does not include actual workload (actual script) the client is supposed to execute. Once a client receives a message about a new remote execution job, it queries the {SmartProxyServer} for actual payload. As the client executes the job, it periodically sends outputs of the job to the capsule, further increasing the number of requests to the {SmartProxyServer}.
+
+These requests to the {SmartProxyServer} together with high concurrency allowed by MQTT protocol can cause exhaustion of {SmartProxyServer} available connections and some requests might fail, making some child tasks of remote execution jobs to hang. This also depends on actual job payload, as some jobs are causing additional load to {ProjectServer}, making it compete for resources if you are using integrated {SmartProxyServer}.
```

Comment:
It's generally "to {SmartProxyServer}" instead of "to _the_ {SmartProxyServer}".

Reply:
Thanks!